core/store/src/db.rs

```diff
@@ -49,22 +49,66 @@ impl Into<io::Error> for DBError {
 #[derive(PartialEq, Debug, Copy, Clone, EnumIter, BorshDeserialize, BorshSerialize, Hash, Eq)]
 pub enum DBCol {
     /// Column to indicate which version of database this is.
+    /// - *Rows*: single row [VERSION_KEY]
```

Comment:
Also, there is no documentation, which column is `is_rc()`.

Reply:
That's a good point - how do we do reference counting in our storage? (we can add it to our documentation in a separate PR)

Reply:
Yes, it's a topic for another discussion. Not blocking this PR.